assets/dev/js/editor/controls/popover-toggle.js

```diff
@@ -17,6 +17,13 @@ export default class ControlPopoverStarterView extends ControlChooseView {
 		} );
 	}
 
+	onShow() {
+		this.popover = this.$el.next( '.elementor-controls-popover' );
+
+		// Attach the current control as a toggle of its popover.
+		this.popover[ 0 ].dataset.popoverToggle = `elementor-control-default-${ this.model.cid }`;
+	}
```

Comment:
@matipojo 

I use it in order to make it easier (and doable...) to select the popover and its toggle in the tests.